test/fuzz/test_runner.py

```diff
@@ -152,6 +167,35 @@ def main():
         )
 
 
+def generate_corpus_seeds(fuzz_pool, build_dir, seed_dir, targets, n):
+    """Generates new corpus seeds.
+
+    Run {n} {targets} at a time without input, and outputs the generated corpus
+    seeds to {seed_dir}.
+    """
+    logging.info(f"Generating corpus seeds to {seed_dir}")
+
+    def job(command):
+        logging.debug("Running {}..".format(" ".join(command)))
+        logging.debug("Output: {}\n\n".format(
+            subprocess.run(command, check=True, stderr=subprocess.PIPE).stderr
```

Comment:
is stderr bytes or an utf-8 string?

Reply:
Bytes :(
Will pass encoding to `run()`.